vm/test.c

```diff
@@ -130,10 +138,14 @@ int main(int argc, char **argv)
 
     uint64_t ret;
 
-    ret = ubpf_run(vm, mem, mem_len, jit);
+    ret = ubpf_run(vm, mem, mem_len, jit, &inst_cnt);
 
     printf("0x%"PRIx64"\n", ret);
 
+    if (inst_cnt.enable) {
+	printf("\n\n instructions (vm/bm) = \n");
```

Comment:
We're missing the value of the instructions in the print.

Also, indentaion is wrong (tab instead of spaces)